packages/snjs/lib/application.ts

```diff
@@ -543,12 +546,9 @@ export class SNApplication {
   }
 
   public setNotesDisplayOptions(
-    tag?: SNTag,
-    sortBy?: CollectionSort,
-    direction?: SortDirection,
-    filter?: (element: SNNote) => boolean
+    search:SearchPayload
   ) {
-    this.itemManager!.setNotesDisplayOptions(tag, sortBy, direction, filter);
+    this.itemManager!.setNotesDisplayOptions(search.tag, search.sortBy, search.isReversedSort, (note:SNNote) => {return notePassesFilter(note,search.showArchiveOrTrashed,search.hidePinned,search.searchTerm)});
```

Comment:
I think this call deserves to be split over multiple lines! Right now it's a little hard to read. We generally try to keep the line width to 80 characters (although I've seen it go over that limit, but it's a decent rule of thumb).

Reply:
`setNotesDisplayOptions` should take a `SearchQuery` param. No need to deconstruct the object before passing it in.